index.js

```diff
@@ -1,10 +1,16 @@
-import {NativeModules, DeviceEventEmitter} from 'react-native';
+import {NativeModules, DeviceEventEmitter, AppState, Platform, PushNotificationIOS} from 'react-native';
 
 const eventsMap = {
     refreshToken: 'FCMTokenRefreshed',
     notification: 'FCMNotificationReceived'
 };
 
+let currentAppState = 'active';
```

Comment:
Yeah we could using AppState but I needed it to be initialized to active for it to work correctly. In iOS currentState can be null on first start.